tests/test_types.py

```diff
@@ -79,6 +80,60 @@ def test_constrained_bytes_too_long():
     ]
 
 
+class ConListModelMax(BaseModel):
```

Comment:
please move model creation into tests since this is part of the test.

Reply:
It is used in multiple tests.. 

I copied this format from other Types..

Reply:
Yes that's a mistake. Should be defined each time.

Reply:
Done